<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ASW Details</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>0</y>
      <width>415</width>
      <height>172</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Počítej!</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Odpor zátěže [Ω]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Reaktance zátěže [Ω]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Zkracovací činitel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="E_f">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="E_zkr">
       <property name="text">
        <string>0.83</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="E_Z0">
       <property name="text">
        <string>75</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Odpor vedení [Ω]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Frekvence [MHz]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Výstupní impedance:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="E_Zlr">
       <property name="text">
        <string>50</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Délka vedení [m]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="E_Zli">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="E_l">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="l_vysledek">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="l_vlna">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="l_swr">
       <property name="toolTip">
        <string/>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="text">
        <string>ČSV</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="toolTip">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
